.github/workflows/tests.yaml

```diff
@@ -0,0 +1,30 @@
+name: Test
+on: push
```

Comment:
I'd refine this to run only on pull requests or pushes targeted to main, otherwise, every pull to every repo will trigger (too much cost and runners competing)

```yaml
on:
  pull_request:
    branches: [ main ]
  push:
    branches: [ main ]
    tags:
      - 'v*.*.*'
```